MAUI/ImageEditor/migration.md

```diff
@@ -516,4 +516,13 @@ Applicable for crop view selection, shape and text annotation selection.
 <td>{{'[ImageEffect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ImageEffect_Syncfusion_Maui_ImageEditor_ImageEffect_System_Nullable_System_Double__)'| markdownify }}</td>
 <td>Method to apply various visual effects to the image, such as adjusted brightness, blur, contrast, exposure, saturation, sharpen, hue, and opacity.</td>
 </tr>
-</table>
\ No newline at end of file
+</table>
+
+## Upcoming features
+
+* Support to browse image from local device storage.

```

Comment:
* Built in support to browse image from local device storage.
* Built in support to save image using folder picker in device storage.
* Localization support for the image editor toolbar, allowing you to use it in your preferred language.
* Support for annotations Z ordering.
* Support for serialize and deserialize the annotations. 
* Support to add custom shapes or views. 
* Support for pan and zoom improvements while performing the crop action on images.
* Support to toolbar customization.

Reply:
Updated a upcoming features